cowin_notifier/config.py

```diff
@@ -1,9 +1,32 @@
+from typing import List, get_origin
+
+from environs import Env
 from pydantic import BaseSettings
+from pydantic.main import ModelMetaclass
 
 
-class Config(BaseSettings):
+class EnvironmentVariable(BaseSettings):
     SLACK_WEBHOOK: str
     MIN_AGE_LIMIT: int = 45
 
 
+class Config(EnvironmentVariable):
```

Comment:
Why do we need this?
Our env variables are pretty straightforward without any complex datatypes
I think `environs` is an overkill, `Pydantic` `BaseSettings` suffice our needs

Reply:
My bad, didn't know `BaseSettings` could be used like that 🙈. Pushed a fix.